__tests__/app.test.js

```diff
@@ -109,6 +109,50 @@ describe("get article by id", () => {
       });
   });
 });
+
+describe("get comments by article id", () => {
+  it("200: returns comments for given id", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const comments = body.comments;
+        expect(comments).toBeSorted({
+          descending: true,
+          key: "created_at",
+        });
+      });
+  });
+  it("404: returns message if article has no comments", () => {
+    return request(app)
+      .get("/api/articles/10/comments")
+      .expect(404)
+      .then(({ text }) => {
+        expect(text).toBe("Path not found");
```

Comment:
When an article is valid and exists in our database but doesn't have any comments we can actually consider that a happy path. The user hasn't done anything wrong, we haven't done anything wrong, no one's commented on the article is all. 

So for this case I'd expect an empty comments array and a 200 status to come back

Reply:
sidenote: really glad you even spotted that this was a scenario